packages/AlgernonTests.package/ALGTestItemList.class/instance/testTabExecutesSingleActionResult.st

```diff
@@ -0,0 +1,14 @@
+testing
+testTabExecutesSingleActionResult
+
+	| browserCount browsers |
+	
+	browserCount := (self findSystemWindowsByLabel: 'Preference Browser') size.
+
+	self runQuery: 'Preferences'.
+	
+	self algInstance keyStroke: (ALGFakeKeyboardEvent new keyString: '<tab>' shallowCopy;
+			 isKeyUp: true).
+			
+	browsers := self findSystemWindowsByLabel: 'Preference Browser'.
+	self assert: browsers size > browserCount
```

Comment:
Not sure if that will be a problem, but if you disable the introduced setting for the tab execution in the image, this and the next test fail (at least in my image). What about two branches of assertion (so if the setting is enabled then assert like you do now and when disabled then assert that no new browser gets opened)? 🤔

Reply:
I think we should move that to another test case. Because the CI always runs the tests isolated, the setting should not be a problem. However I guess it would be convenient to ensure the setting is set / set it and restore it to be able to run it locally successfully.

Reply:
Should be solved with the most recent commit